### ISSUE TITLE
Bridge: Update shipyard for project

### ISSUE THREAD

**Issue by laneli** [keptn/keptn#4618]
## User story
As a user, I want to update the shipyard file for my project directly from the bridge.

### Description
Re-use the shipyard drag and drop introduced with the create screen to update the shipyard file.
Everything should be prepared with #4493 what is missing is the API call.

### Acceptance criteria
- [ ] The dropdown is enabled also for project settings
- [ ] "Upload shipyard" button has functionality and the API call is made
- [ ] Error and success handling in case the upload does not work

### Definition of done
- [ ]  Everything in acceptance criteria implemented and manually tested
- [ ]  Added E2E Test
- [ ]  Documentation updated

**Comment by johannes-b** [comment]
Before starting with this issue, please verify if /PUT project endpoint contains checks if no stage has been added/deleted.

**Comment by johannes-b** [comment]
I have put the `on hold` label on this issue because the shipyard (and the way of working with the shipyard) is subject to change when the KEP: https://github.com/keptn/enhancement-proposals/pull/67 becomes real.
